tests/chainer_tests/functions_tests/loss_tests/test_decov.py

```diff
@@ -6,12 +6,12 @@
 import chainer
 from chainer.backends import cuda
 from chainer import functions
-from chainer import gradient_check
 from chainer import testing
 from chainer.testing import attr
+from chainer.testing import backend
 
 
-def _deconv(h):
+def _decov(h):
     h = cuda.to_cpu(h)
```

Comment:
I think that `forward_expected` only gets numpy arrays so this call may not be necessary.

Reply:
Nice catch!